tests/test_create_save_summary.py

```diff
@@ -198,13 +192,12 @@ def test_generate_coding_summary(reads, data_folder, single_alphabet_ksize_true_
         reads, True, True, True, "bloom_filter.nodegraph", "protein", 7, 0.5
     )
     test_summary = create_ss.generate_coding_summary(single_alphabet_ksize_true_scores)
-    print(test_summary)
     true_summary = {
         "input_files": ["SRR306838_GSM752691_hsa_br_F_1_trimmed_subsampled_n22.fq"],
         "jaccard_info": {
-            "count": 44.0,
+            "count": 44,
             "mean": 0.085830733808675,
-            "std": 0.25321503253861455,
+            "std": 0.2503210514088884,
```

Comment:
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.std.html - pandas normalizes by n-1 
and numpy by N - https://numpy.org/doc/stable/reference/generated/numpy.nanstd.html#:~:text=Compute%20the%20standard%20deviation%20along,otherwise%20over%20the%20specified%20axis.